samigo/samigo-app/src/webapp/jsf/questionpool/editPool.jsp

```diff
@@ -188,7 +188,7 @@ function textCounter(field, maxlimit) {
   <f:param name="qpid" value="#{questionpool.currentPool.id}"/>
   <f:param name="outCome" value="editPool"/>
 </h:commandLink>
-<h:outputText rendered="#{questionpool.importToAuthoring != 'true'}" value=" #{questionPoolMessages.separator} " />
+<h:outputText rendered="#{questionpool.importToAuthoring != 'true' && questionpool.canAddPools}" value=" #{questionPoolMessages.separator} " />
```

Comment:
the first boolean uses != 'string' and the second boolean doesn't .. i would just be consistent. i prefer `questionpool.importToAuthoring eq false && questionpool.canAddPools eq true`

Reply:
Hi @ottenhoff , I’ve just added a new commit addressing your request to ensure consistency in the boolean logic. Since I had originally followed the format used in the rest of the documents, I’ve made the same adjustments across the board for consistency.

Reply:
why compare them as strings though if they are booleans? i know the old code did it, but the goal is to always improve

Reply:
updated code